### PR TITLE
images: Disable PerSourcePenalties on fedora-41

### PR DESCRIPTION
**Pull request by mvollmer**
Just like for Arch and RHEL 10.

 * [x] image-refresh fedora-41

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh fedora-41 done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-41-20241114-144038

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-fedora-41-11417d53-20241114-142218/log.html

File: images/scripts/fedora.setup [hunk 1]
```diff
@@ -228,6 +228,12 @@ dnf repoquery --installonly --latest-limit=-1 -q | xargs --no-run-if-empty rpm -
 # pmlogger.service sometimes causes long shutdown hangs; disable all PCP services and timers
 systemctl --all --legend=false list-units 'pm*' | awk '{print $1}' | xargs systemctl disable
 
+if [ "${IMAGE#fedora-41*}" != "$IMAGE" ]; then
+  # Disable PerSourcePenalties, they interfere with the rapid failed
```
Comment (martinpitt) [inline images/scripts/fedora.setup:232]: OK! Let's generalize this once we move to 42, and then just skip it for 40.

**Comment by mvollmer** [comment]
The bot isn't botting.

**Comment by cockpituous** [comment]
image-refresh fedora-41 done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-41-20241114-153926

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-fedora-41-e63938ef-20241114-152130/log.html

**Comment by cockpituous** [comment]
Failed. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-fedora-41-11417d53-20241114-144611/log.html

**Review by martinpitt — APPROVED** [review]
Cheers!

**Pull request merged**